biblibre/urungi
# Replace app-list-panel by <table>

Closes: #65

Test plan:
1. Go to the lists of layers, reports and dashboards, and play with column ordering and filtering, pagination, and check that buttons work too
2. Go to dashboard edit page and try to import a report
